package.json

```
@@ -54,11 +54,12 @@
     "build:clean": "rimraf dist",
     "build:code": "esno ./scripts/bundle.ts",
     "build:types": "tsc --emitDeclarationOnly --outDir dist/types",
-    "build": "run-s build:clean build:code build:types",
+    "build": "run-s build:clean build:code build:types docs:api",
     "docs:build": "vitepress build docs",
     "docs:build:ci": "run-s build docs:build",
```

I think we just need to move the `docs:api` to here and remove it from `build`

```suggestion
    "docs:build:ci": "run-s build docs:build docs:api",
```

---

In addition to that we could document in e.g. the CONTRIBUTING.md that this `docs:api` needs to be runned once to prepare the `docs:dev`

---

Or add two script like

```
"docs:dev": "run-s docs:dev:prepare docs:dev:run",
"docs:dev:prepare": "run-s build docs:api"
"docs:dev:run": "vitepress dev docs"
```

But 
1. it's a bit verbose
2. it would always run `build` and `docs:api` before starting the dev server, also if not need :thinking: